_toc.yml

```
@@ -0,0 +1,17 @@
+format: jb-book
+root: README
+parts:
+  - caption: Relevant Research Papers
+    chapters:
+    - file: docs/Relevant_papers.md
+  - caption: Getting Started
+    chapters:
+    - file: notebooks/getting_started_notebook
+    - file: pyxis_mapping
```

```suggestion
    - file: notebooks/pyxis_mapping
```
The notebook isn't rendering for me without this prefix